web/client/localConfig.json

```diff
@@ -787,7 +837,38 @@
       {"name": "MapType"},
       {"name": "CreateNewMap"},
       {"name": "Maps"},
-      {"name": "Examples"}
+      {"name": "Examples"},
+      {
```

Comment:
If you want 2 different default configurations for home and map, use something a "preset": <home>|<map> you can easily override with `localConfig.json`.
You can save that preset as separate files in the plugin folders. rawSteps configuration will override the steps in the preset.